packages/ui/GridListImageItem/GridListImageItem.less

```diff
@@ -3,9 +3,7 @@
 
 	.icon {
 		position: relative;
-		display: block;
-		margin: 0;
-		opacity: 0.5;
+		display: none;
```

Comment:
Why did we change this to `display: none` when the icon is already conditionally rendered?

Reply:
I guess my observation was that by setting `display: none` here, we require a theme set the `display` in their CSS for `.icon` to be usable. That doesn't seem like the right default to me.

Reply:
I'll take it out.